### ISSUE TITLE
maybe there is a mistake 

### ISSUE THREAD

**Issue by mygh1209** [dotnet/docs.zh-cn#473]
"public Prop {" should be "public string Prop" at the the topic of Properties Accessibility

---
#### 文档详细信息

⚠ *请勿编辑此部分。 docs.microsoft.com ➟ GitHub 问题链接必须具有此部分。*

* ID: d94b01ab-fe87-23fc-44b6-da9f3f627bd5
* Version Independent ID: a86008b1-2fdd-bbb9-3bbf-8fc8c51c1b71
* Content: [类Classes - C# language specification](https://docs.microsoft.com/zh-cn/dotnet/csharp/language-reference/language-specification/classes#properties)
* Content Source: [spec/classes.md](https://github.com/dotnet/csharplang.zh-cn/blob/live/spec/classes.md)
* Product: **dotnet-csharp**
* Technology: **csharp-spec**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **wiwagn**

**Comment by srvbpigh** [comment]
Hello, @mygh1209
 
Thank you for your feedback.
 
We are actively reviewing your comments and will get back to you soon.


Kind regards,
Microsoft DOCS International Team

**Comment by mairaw** [comment]
Thank you @mygh1209. I've opened a PR at https://github.com/dotnet/csharplang/pull/2790 to fix this.

**Comment by CeciAc** [comment]
Hello, @<contributor's GITHUB name>,

Your contribution is much appreciated. Our team reviewed your feedback and it appears that this issue has been already logged.
Please review issue https://github.com/dotnet/csharplang/pull/2790 and https://github.com/dotnet/csharplang/pull/2789
For this reason we are closing this issue since it is duplicated.

Kind regards,
Microsoft DOCS International Team
